src/Util/Logger/Monolog/FriendicaIntrospectionProcessor.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Friendica\Util\Logger\Monolog;
+
+use Friendica\Util\Introspection;
+use Monolog\Logger;
+use Monolog\Processor\ProcessorInterface;
+
+/**
+ * Injects line/file//function where the log message came from
+ */
+class FriendicaIntrospectionProcessor implements ProcessorInterface
```

Comment:
Why did you include `Friendica` in the class name? It already is in the `Friendica\Util\Logger\Monolog` namespace.